libpod/options.go

```diff
@@ -305,6 +305,18 @@ func WithSELinuxLabels(processLabel, mountLabel string) CtrCreateOption {
 	}
 }
 
+// WithUser sets the user/group to run with the container
```

Comment:
Can you mention that it's set as user:group

Reply:
Done